### DISCUSSION TITLE
KTON DAO

### DISCUSSION THREAD

**Discussion by SasoLithops** [ringdao-community-guild/discussion#32] [Ideas]
Hello everyone!

I am starting a discussion about having a subDAO.

What do you think after @hackfisher proposed this first having a KTON DAO?

My ideas so far:

We could ask for minimum hold of 20 KTON to join the KTON DAO. 

It would be nice if we could also have a membership NFT (for start I can make them free from DR.CAO AI). We could ask for 3 KTON for this NFT 100% going to KTON treasury.

In future we could asap discuss giving more utility to KTON.

I have helped as member of ION community launch IBCX index for Cosmos ecosystem.  So my thought is we can also have index of POLKADOT tokens(I don't  think there is one yet) but this would need extra team from outside or if Darwinia treasury(it could be build on Darwinia network) and its devs are willing to participate. But it could be done from KTON DAO and KTON should have some small weight in index too. 

So what do you think?

**Comment by SasoLithops** [comment]
I just thought of a number 3😊 But I agree on your point, utility first. Then we discuss exact number.

**Comment by TobiasAlfred** [comment]
Super idea, make it happen, bro.

**Reply by SasoLithops** [reply]
We are going to make it happen together 😉

**Comment by ahjxcrz** [comment]
Thank you Saso for proposing this great idea! The idea of launching a Polkadot token index sounds intriguing, and given your experience with IBCX in Cosmos, it would be nice to make a product around this idea, and get Darwinia team involved (maybe some cross-chain features with XCM? or integrated with the game? ). There should be no bar to join the DAO, but there should be bar to vote in the DAO, so maybe anyone interested in the DAO can join but need to hold 20 KTON or the NFT to get involved?  And also, the utility would be important for this one, we should have some more discussions and maybe organize some AMAs like twitter space? In addition, the DAO can partnership with other projects to get them involved in the index, at least my guild would be happy to get involved, we can even create a channel there and discuss everything and recruit members.

**Reply by SasoLithops** [reply]
Thank you. These are great points imo. Sounds great to me no bar to join DAO and 20 KTON to vote👍

Imho I think it would be great to partner with your guild. You are very respected in Polkadot and Kusama community and do great job on keeping us updated. I also trust your knowledge and integrity. It would be great to find more proactive members to push this forward.

I am all for. Let's see what others have to say.

**Comment by SasoLithops** [comment]
Hello Kton hodlers!

BIG THANK YOU TO ALL THAT PARTICIPATED SO FAR🏆

I have done some more thinking.

I have just contacted Darwinia Nada for help. I would like to get voting page on track. So we start voting on some ideas.

1. NAME:

I was thinking  of changing  it to KTON DIAMOND HANDS DAO (KDH short)

Any other proposal for name is warmly welcome,so we can vote on it.

2. NFT:

What do you think of creating OG Kton Dao NFT? It will be given to Team and those who supported Darwinia in bear market (TG chat, Discord, Twitter...) Also to honest validators. 
Let me know what you think.


3. INTRODUCTION FORM


I would like to make this DAO more personal, well not KYC personal😅 Just have some questionare ready so that we all learn what someone is good at.

I thought of:

- What do you do in RL?
- Your hobbies?
- Do you have any skills to help Kton DAO( no worries if you don't)
- What do you think should be KTON DAO purpose?
- How to add more value to KTON?

Please add your questions.


I would love to see you participate. Instead of just us pushing all expectations on team,we can help to build future together.

**Reply by sonoferin2000** [reply]
what would purpose of kton dao be? how do you all feel about  the name KTONPULSE? i think it has a nice ring to it. that's my recommendation. kton isn't my choice for token name,but that's the cards we were dealt. but kton was issued to show our commitment to the project,  pulse conveys the idea of life, like a heartbeat., combination of kton and pulseimplies a lively and continuous commitment, we are actively engaged,showing continued dedication and involvement in the project..anyways, I'm open toany other suggestions as well.
so maybe no minimum amount to join, if you got any kton,you should be able to have a say should be open to all, if someone has a good idea, he should be able to express it through vote. but maybe have a minimum staked balance to actually propose anything.if their ideas have enough merit, it could be sponsored by others and included for vote, but 1 kton= 1 vote,sounds fair, its weighted more towards people who have locked their tokens for the long haul. it bars nobody, they still have a voice,but gives those who believe in the project  more of a voice. i don't think we should have to pay for nft membership,  especially not in kton. our staked kton balance should be membership card enough.  I've been diamond handing kton since it first released.  what utility would nft have? reduced fees within the ecosystem maybe? a nice plot of land in the game? a portion of any slashing events,or bridge fees etc being directed our way  maybe,if i remember right,that was how it was supposed to workinitially, kton holders would get a potion of fees and stuff,but it never was implemented.  already have a ton of darwinia polkapets with nothing to show for them, didn't get ring drop.So basicly I'm very reluctant of buying more nfts. these nfts hurt me bad, this idea should be fleshed out a lot more before committing to dao.  needs to have solid purpose. would we have impact in the direction of darwinia project, or would it just be over kton? id like to hear others input, i don't think we should rush to start a dao just for the sake of doing it,but the idea of haveing some voice in matters is attractive.

**Reply by SasoLithops** [reply]
Hi!

I only know one Son of Erin with many Polkapets and it is Ryan;-) Am I right? 

Initial dea was/is to work on Kton utility bring team in discussions to let Kton get more value and usecases. But all is up to discussions. I actually thought discussions here would help us pave the "whitepaper". There are still some of us who have held to Kton(definitely agree on name:-)) )like you said  through all and I feel like it was completley forgoten since it was invented in EL as commitment token and I wanted a group of us to help in providing ideas that may potentialy reinburst Kton.

If KTONPULSE has nice RING to it sounds great;-) I like it the way you describe it even more.

Yes just holding Kton to join is better. When we get to governance it would be quite complicated to lock Kton for proposal posting as Kton already locked. Maybe it could be like in polkadot double locked? Also in governance I would like to avoid whales overvoting small holders. Maybe(just idea) we could have some tier points from 1-3:

1 vote. small ammount <5 staked, <1 month staked
2 votes medium ammount 5-50, < 6 months 
3 votes high ammount >50, more then 6 months

so no whales can take over governance.

I agree no need to buy NFT. Was bad idea. I wanted to give KTON another usecase, but was poor idea.

True on we need idea flashed out. That is why I tried starting this discussion. No commitment as of yet. We need to set solid ground to build on. I also hope more people join debate and take time and give opinion like you did. 

To add I really liked idea it was at first that developers working on Darwinia should for usage of tech pay in KTON. That got me totally bullish as we knew Kton would be hard to get. So starting a DAO with solid ground to later build on with transparent governance to give Kton more utilities is end goal imho. 

I thank you again for very solid points!

**Comment by sonoferin2000** [comment]
My name is Joe, but it kind of rhymes with Ryan I  guess, close enough.
I agree that putting KTON to good use is essential, and while staking is a solid option, I believe we can explore alternatives to spending KTON tokens on NFTs at this moment. I've been thinking about a way for people to contribute to capitalizing the KTONPulse DAO without experiencing any significant loss.
My idea involves starting our own validator node, deploying it on the network, and directing all rewards generated to the DAO treasury. we would set fee at 100% but it would be for a cause,not greed. Participants would create a secondary wallet to stake a small amount specifically for this purpose, while they can continue staking their main stack as usual to earn rewards on other nodes. 
To express our gratitude for their voluntary contribution to the DAO's benefit, we would issue them an NFT as a token of appreciation. This NFT could also function as a membership card within the DAO.
It's a great idea to consider alternative streams to fund the treasury as it grows. Developing some simple apps that not only benefit the network but also generate value for the DAO is a forward-thinking approach. These primitive apps could serve as both a service to the community and a means to sustain the DAO financially. I do have a few ideas in mind if we could get help building them out and deploying them.

**Comment by sonoferin2000** [comment]
I recommend reaching out to Polkapets to explore the possibility of deprecating their pets on the Ethereum mainnet. The proposal involves taking a snapshot of addresses holding these pets, along with their quantities, and subsequently reissuing them on Darwinia. Currently, these pets are underutilized on Ethereum due to the high cost of transactions, hindering their market activity.

By bringing these pets to Darwinia, we could potentially revitalize their utility and attract a broader audience. As a project specializing in building bridges within the Polkadot ecosystem, Darwinia serves as an ideal network to host these pets. This move not only aligns with the mission of fostering collaboration but also contributes to the growth of both Polkapets and Darwinia by creating a more accessible and vibrant environment for these unique assets.

We can create a simple NFT trading app, allowing two addresses to conduct direct trades. For NFT-to-NFT transactions, we could introduce a flat fee in KTON or Ring tokens. In the case of trades involving fungible tokens, a small percentage of the transferred tokens could serve as a fee, contributing to our DAO treasury. This service would provide significant value to the community, enabling larger trades without adversely affecting liquidity pools.

Moreover, we can explore developing additional basic apps with the goal of generating new funding streams for our DAO treasury. These efforts aim to not only enhance community engagement but also ensure a sustainable financial foundation for our DAO, allowing us to continue providing valuable services and support to our community members.

**Comment by sonoferin2000** [comment]
I believe that bringing Polkapets to Darwinia could be a significant opportunity for our ecosystem. This move not only promotes our name to other projects and their user bases but also entails onboarding them onto our network. As they transition, they will need to bring funds to transact, expanding our user base and potentially providing additional financial resources to our ecosystem. This strategic approach opens up new avenues for collaboration and growth, establishing a stronger presence for Darwinia within the broader blockchain community.

**Comment by sonoferin2000** [comment]
Exclusive Airdrops: Provide NFT holders with exclusive airdrops of new tokens or other digital assets associated with projects supported by the DAO.

    Early Access: Grant NFT holders early access to new features, products, or projects developed or funded by the DAO. This could include beta testing opportunities or early previews.

    Collaboration Opportunities: Offer NFT holders the chance to collaborate with projects that the DAO is involved in. This could involve joint ventures, partnerships, or special collaboration events.

    Whitelist Spots: Reserve whitelist spots for NFT holders in upcoming token sales or NFT drops associated with projects aligned with the DAO's mission.

    Governance Influence: Consider giving NFT holders enhanced governance rights within the DAO. This could involve voting power, proposal submissions, or other forms of direct participation in decision-making processes.

    Exclusive Merchandise or Collectibles: Create exclusive merchandise or digital collectibles that are only available to NFT holders.

    Educational Resources: Provide NFT holders with educational resources or workshops related to blockchain, cryptocurrency, or the specific focus areas of the DAO.

    VIP Events: Grant NFT holders access to VIP events, webinars, or conferences organized by the DAO or its partner projects.

    Loyalty Programs: Implement a tiered loyalty program based on the duration or amount of stake delegated by NFT holders, offering increasingly valuable rewards for long-term commitment.

**Comment by SasoLithops** [comment]
Mr. Rhymes with Ryan alias Joe hi again!:-)

Let me reply my opinion.

1. On nodes. To be honest I like the idea. However I have some concerns.

- It would be a question who would run it? 
- The monthly cost for month if I remember corectlly is cca 60$. This would have to be taken out of reward to pay the node provider for basic expenses. 
- Also we need Power around 30M to guarantee us making it to active set. This is quite a lot and hard to reach with 100% commision.

2. As for Polkapets. They seem dead. I know they moved new collections to Polygon. They look also dead. Also the very unresponsive, I saw your conversation with them on their Discord. I also fear it would take at least one full time employed developer to deal with all the pets. Not to mention gzillons of unhappy customers. I have forgotten about them although they were not cheap but no decent reply from their team. 

3. We alredy had an nft marketplace for Crab in Raregems but just checked no longer supports Crab.

4. As for your  last thread( https://github.com/orgs/dcdao/discussions/32#discussioncomment-7743681 )I like all of the proposed. They all have potential.

Let me know what you think and if I missed some important points. Hope more join to comment.

**Reply by sonoferin2000** [reply]
Exploring a node hosting service could be a viable option for us. While the idea of a fixed 100% commission is open for discussion, it's paramount that everyone comprehends the core purpose of this node – benefiting the DAO. This could provide an opportunity for contributors to participate without significantly impacting their balances. I don't expect anyone to stake their entire holdings on this node, but if we can encourage individuals to allocate a portion of their holdings to a secondary wallet earmarked for this purpose, achieving the 30 million power becomes feasible.

In addressing the requirement for 30 million power to secure an active spot, strategic planning is essential. By proactively announcing our intentions well in advance, disseminating information about our objectives, and rallying support, we might justify the expenses associated with setting up a node. This proactive approach could make the target more attainable and set the stage for successful community involvement.

Regarding Polkapets, it was initially considered as a potential method for onboarding people and promoting our project. However, after careful consideration, I've decided to set this idea aside.

On the topic of my suggestion, it wasn't geared toward creating a marketplace. Instead, it aimed to facilitate trustless transactions between two parties privately. This system ensures that both parties fulfill their end of the bargain without the risk of one party running off with the tokens. Whether it's a trade of NFTs or fungible tokens, this method eliminates the need for one party to send their item first, reducing the risk associated with the exchange. While it doesn't replace the need for a full-fledged marketplace, it provides a secure option for individuals who are already in contact and want to transfer value without the risk of slippage due to small liquidity pools. Small apps like this can significantly benefit the community.

Regarding the treasury, I acknowledge that I frequently mention it. As we progress, funding will become a necessity. Whether it's compensating an artist for designing NFTs or engaging someone to build a smart contract, these endeavors require resources. While some individuals might offer their services at no cost initially, sustained efforts often involve expenses. Having a funded treasury would empower us to make decisions without the burden of worrying about how to finance essential elements of our projects, enabling smoother and more informed decision-making.

**Reply by SasoLithops** [reply]
I still think ( now it is cca 33M Power to make it to active set ) getting to active set will be hard. For that we would need cca 6.5M Ring and 4.2k KTON ( Or any other ratio of these two I callculated on my holdings)  which I think is very hard to achive with high fees. As you see participation is low at this point for many people to give up on their rewards.
However if we set commision to 10%-15% which is attractive and state as you said in other reply to David solid grounds for KTON DAO we can attract more users there imho. Not just us.

As for a dApp for " trustless NFT transfers between two parties cross chain" that is actually very good idea using Darwinia tech that could be done(in future) but would need huge ammount of development, don't think and dApp even does that.

I absolutely agree on the part we do need a healthy self funding treasury. However long way till there. First we need to set the path. And attract more members imho.

**Comment by sonoferin2000** [comment]


**Reply by sonoferin2000** [reply]
It's essential to have clear documentation, such as a mission statement, bylaws, and other foundational documents, to establish the framework and purpose of the DAO. If there's no information available on these aspects, it would indeed be challenging for members to align on the goals and vision of the organization.
To move forward effectively, it would be advisable for the DAO to establish and communicate its foundational principles, governance structure, and mission. This could involve drafting and adopting bylaws, creating a mission statement, and providing transparency on the organizational structure.

**Reply by SasoLithops** [reply]
You both gave great points. I also think that needs to be a start to develop into healthy DAO.

**Reply by sonoferin2000** [reply]
Partnershipdude emphasized the non-profit nature of our endeavor, yet raised the question of mutual benefit, creating a potential tension. To address this, I propose we collectively define the purpose of our DAO, aligning our goals and aspirations. Given the limited active members, let's streamline our focus by openly discussing individual expectations and collaboratively crafting a concise mission statement. This will serve as a foundation for unity and clarity, helping us navigate towards our shared objectives. Let's work together to establish a common understanding of what we hope to achieve with this initiative.

**Reply by SasoLithops** [reply]
Sorry for late reply, me on holidays. I agree I will post my expectations in one week time when I get home, please guys share yours in the mean time if you not busy during holidays 😉

**Reply by sonoferin2000** [reply]
Personally, my key objective in this DAO is to discover opportunities that contribute to the long-term value enhancement for committed KTON stakeholders. I strongly believe that introducing extra benefits for individuals engaged in KTON staking has the capacity to greatly increase the demand for the token, subsequently resulting in a rise in the market price of KTON. I advocate for the DAO to actively explore creative strategies that deliver tangible value to stakeholders. Specifically, I am enthusiastic about the idea of implementing revenue sharing to kton stakers and believe that we should take proactive steps to turn it into a reality.

**Reply by sonoferin2000** [reply]
it doesnt look like there is any interest at all on this.

**Reply by SasoLithops** [reply]
Sorry it slipped my mind. It looks like there is no interest(at this time). And it is hard to push forward with little or no engagement. If you however have energy to push this forward you can count on me for help.

**Comment by sonoferin2000** [comment]
its not you,its just in general,thread has been open since july 29th,
theres just no interest.

**Reply by SasoLithops** [reply]
I agree with you. There is one option though and I am serious(although it might seem dumb). You and I start KTON DAO and start playing PINK games and other oportunities. WE open multisig where we both need to sign for any transfer and start earning pink tokens. We sell those to KTON and stake to start earning Ring. Then slowly we start with ideas as funds accumulate.
I was inspired by your games idea.

**Reply by Laki389** [reply]
> its not you,its just in general,thread has been open since july 29th, theres just no interest.

I think this topic would have been better suited to https://github.com/orgs/darwinia-network/discussions
and maybe there it would have received more attention from the Itering developers.

**Reply by SasoLithops** [reply]
It is fine here. Darwinia devs already aware since august last year😀 We can get the help we need if we get to that.![IMG_20240130_064146.jpg](https://github.com/dcdao/discussion/assets/101176902/c008ad37-8b96-4ac0-9352-91650fc4295d)

**Comment by SasoLithops** [comment]
@sonoferin2000 

Check here reply from @hackfisher 

https://github.com/orgs/darwinia-network/discussions/1393#discussioncomment-8271744


![IMG_20240131_175608.jpg](https://github.com/dcdao/discussion/assets/101176902/7a6bd717-9860-4f6a-8e97-0f9ff5ed4930)

We can build on this too. Members have to have good intentions. We need to be careful for people might start take advantages.

**Reply by SasoLithops** [reply]
This is another great usecase for KTON.

https://github.com/orgs/darwinia-network/discussions/1397

**Comment by hackfisher** [comment]
Latest proposal to this topic is requesting for comment:

https://github.com/orgs/darwinia-network/discussions/1512

**Comment by SasoLithops** [comment]
@sonoferin2000 Please check and join in debate link above

**Comment by sonoferin2000** [comment]
Im glad to see that this is gaining momentum, just got back from
vacation,but will try to give my thoughts once i get home from work, im
operating on just a couple horse of sleep now

**Comment by SasoLithops** [comment]
You are one of first to brainstorm and alliked the idea of KTON DAO. Please read an let me know what you think.

@ahjxcrz @sonoferin2000 @TobiasAlfred 

https://github.com/orgs/darwinia-network/discussions/1566

**Comment by hackfisher** [comment]
https://www.tally.xyz/gov/ktondao
